pv_site_api/main.py

```diff
@@ -355,6 +361,57 @@ def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_sess
     return res
 
 
+async def get_inverters_helper(session, inverter_ids):
+    if int(os.environ["FAKE"]):
+        return make_fake_inverters()
+
+    client = session.query(ClientSQL).first()
+    assert client is not None
+
+    async with httpx.AsyncClient() as httpxClient:
+        headers = {"Enode-User-Id": client.client_uuid}
+        if not inverter_ids.length:
+            return None
```

Comment:
could we change this to be more explicit? 
```suggestion
        if inverter_ids.length == 0:
            return None
```